src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -4917,4 +4964,100 @@ public static RepairRetrySpec getRepairRetrySpec()
     {
         return conf == null ? new RepairRetrySpec() : conf.repair.retries;
     }
+
+    public static int getCmsDefaultRetryMaxTries()
+    {
+        return conf.cms_default_max_retries;
+    }
+
+    public static void setCmsDefaultRetryMaxTries(int value)
```

Comment:
This method seems to be not used and the name does not really match the parameter. Should I remove it?